internal/provider/credential_provider_resource.go

```diff
@@ -333,7 +333,7 @@ func (r *credentialProviderResource) Schema(_ context.Context, _ resource.Schema
 						},
 					},
 					"lifetime_expiration": schema.StringAttribute{
-						Description: "ISO 8601 formatted Lifetime Expiration of the OAuth Authorization Code credentials requested by the Credential Provider.",
+						Description: "ISO 8601 formatted Lifetime Expiration of the OAuth Authorization Code credentials requested by the Credential Provider. This expiration timer begins when the user successfully completes an authorization of the Credential Provider and will be set to the authorization time plus the Credential Provider Lifetime value at that moment.",
```

Comment:
@auzunel-broadangle I updated the verbiage to try and make it clearer - can you also update the same in the Data Source and make sure the docs are updated?